modules/examiner/jsx/columnFormatter.js

```diff
@@ -0,0 +1,51 @@
+/* exported formatColumn */
+
+/**
+ * Modify behaviour of specified column cells in the Data Table component
+ * @param {string} column - column name
+ * @param {string} cell - cell content
+ * @param {arrray} rowData - array of cell contents for a specific row
+ * @param {arrray} rowHeaders - array of table headers (column names)
+ * @return {*} a formated table cell for a given column
+ */
+function formatColumn(column, cell, rowData, rowHeaders) {
+
+  if (loris.hiddenHeaders.indexOf(column) > -1) {
+     return null;
+  }
+  // Create the mapping between rowHeaders and rowData in a row object.
+  var row = {};
+  rowHeaders.forEach(function(header, index) {
+    row[header] = rowData[index];
+  }, this);
+  console.log(rowData);
+
+  if (column === 'Examiner'){
+      var url  = loris.BaseURL + "/examiner/editExaminer/?identifier=" + row["ID"];
+      return <td>
+                <a href ={url}>{cell}</a>
+             </td>;
+     }
+
+  if (column === 'Radiologist'){
+      var radiologist = 'No';
+      if (row['Radiologist']=='1')
+            radiologist = 'Yes';
+      
+      return <td>
+                {radiologist}
+             </td>;
```

Comment:
I think for multiline returns it's best practice to do

`return (
...
);` (with the brackets). That way you can't get bitten by javascript's implicit semicolon injection rules.